web-jehlomat/src/Components/Buttons/PrimaryButton/PrimaryButton.tsx

```diff
@@ -18,10 +19,10 @@ const Button = styled.button`
     padding: 15px 30px;
 `;
 
-const PrimaryButton: FC<Props> = ({ text, ...props }) => {
+const PrimaryButton: FC<Props> = ({ text, fontSize, ...props }) => {
```

Comment:
I would narrow possible size options. Like "S", "M", "L", "XL". fontSize is too much freedom